rs/replicated_state/src/canister_state/queues.rs

```diff
@@ -144,6 +144,12 @@ pub struct CanisterQueues {
     /// and remote subnet senders; as well as within the local subnet senders and
     /// remote subnet senders groups.
     input_schedule: InputSchedule,
+
```

Comment:
GitHub prevents me from commenting on the actual line, but I think the comment regarding the invariants on `CanisterQueues` above may need updating as well.

Reply:
Listed `callbacks_with_enqueued_response` as a hard invariant, thanks for pointing it out.